scripts/update_taxonomies.py

```diff
@@ -18,7 +18,7 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def update_taxonomies(filename):
+def update_taxonomies(filename, warn=True):
```

Comment:
The purpose for the warning argument was to prevent warnings when using the function on a test. Would I be able to set a logger level above warning on my test?